orbit/cmd/orbit/orbit.go

```
@@ -462,6 +477,16 @@ func main() {
 					log.Info().Msg("No cert chain available. Relying on system store.")
 				}
 			}
+
+		}
+
+		orbitClient, err := service.NewOrbitClient(fleetURL, c.String("fleet-certificate"), c.Bool("insecure"))
+		if err != nil {
+			log.Error().Msg("Error Creating Orbit Client " + err.Error())
```

Same here, let's just return the error (and below).